SEC/Securities/SecuritiesListings.rdf

```diff
@@ -207,25 +209,13 @@
 				<owl:someValuesFrom rdf:resource="&fibo-fnd-acc-cur;Currency"/>
 			</owl:Restriction>
 		</rdfs:subClassOf>
-		<rdfs:subClassOf>
-			<owl:Restriction>
-				<owl:onProperty rdf:resource="&fibo-sec-sec-lst;isTradedOn"/>
-				<owl:someValuesFrom rdf:resource="&fibo-fbc-fct-mkt;Exchange"/>
-			</owl:Restriction>
-		</rdfs:subClassOf>
 		<rdfs:label xml:lang="en">listing</rdfs:label>
-		<skos:definition>registry entry for a security that provides information regarding the eligibility for making that security available for trading</skos:definition>
+		<skos:definition>registry entry for a security that is managed by an exchange and that provides registration information regarding the eligibility for making that security available for trading</skos:definition>
```

Comment:
I don't think we should be modeling pure data artifacts such as "registry entry" which will be specific to each Exchange and probably proprietary. All these properties are actually applicable to the Offering which was Listing and is now ListedSecurity. In other words I don't think we need separate classes.